src/main/java/frc/robot/commands/DriveCommand.java

```diff
@@ -211,6 +211,13 @@ public void execute() {
     }
   }
 
+  public boolean isAtSetPoint() {
+    if (this.mode == TargetMode.CORAL_STATION || this.mode == TargetMode.REEF) {
```

Comment:
We should check with Isaac if the color should change just for coral station, or maybe for anything